internal/services/containers/migration/kubernetes_cluster.go

```diff
@@ -1140,6 +1140,11 @@ func (k KubernetesClusterV0ToV1) Schema() map[string]*pluginsdk.Schema {
 			Optional: true,
 		},
 
+		"node_os_channel_upgrade": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+		},
+
```

Comment:
The schemas in migration are point in time references, so we don't need to add new properties here.
```suggestion

```

Reply:
done.